lib/api/core/models/repositories/profileRepository.js

```diff
@@ -271,6 +271,15 @@ class ProfileRepository extends Repository {
 
     return profile.validateDefinition()
       .then(() => {
+        if (profile._id === 'anonymous') {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)